engines/real-time-ensembler/app.Dockerfile

```diff
@@ -0,0 +1,30 @@
+ARG BASE_IMAGE
+
+FROM ${BASE_IMAGE} as builder
+
+ARG MODEL_URL
+RUN gsutil -m cp -r ${MODEL_URL} .
+ARG FOLDER_NAME
+
+# Install dependencies required by the user-defined ensembler
+RUN /bin/bash -c ". activate ${CONDA_ENVIRONMENT} && conda env update --name ${CONDA_ENVIRONMENT} --file /${HOME}/${FOLDER_NAME}/conda.yaml"
+
+# Use conda-pack to create a standalone enviornment
+# in /venv:
+RUN conda-pack -n real-time-ensembler -o /tmp/env.tar && \
+  mkdir /venv && cd /venv && tar xf /tmp/env.tar && \
+  rm /tmp/env.tar
+
+RUN /venv/bin/conda-unpack
+
+FROM debian:latest
+
+COPY --from=builder /ensembler ./ensembler
+COPY --from=builder /pyfunc_ensembler_runner ./pyfunc_ensembler_runner
+COPY --from=builder /run.sh /run.sh
+COPY --from=builder /venv /venv
+
+RUN /bin/bash -c ". /venv/bin/activate && \
+    python -m pyfunc_ensembler_runner --mlflow_ensembler_dir /ensembler --dry_run"
+
+CMD ["/bin/bash", "./run.sh"]
```

Comment:
Can we simply activate conda env by default so all of the commands will be executed in that env already activated? In this case you don't need this extra `run.sh` file. 

This is how it was done in batch ensemble image: https://github.com/gojek/turing/blob/c1303487fde3af8704fa03cf561e969db013530e/engines/batch-ensembler/Dockerfile#L96

Reply:
Okay that's possible too, alright I'll do everything within the `Dockerfile`